planner/src/main/java/com/dask/sql/application/DaskPlanner.java

```diff
@@ -86,4 +103,12 @@ public DaskPlanner() {
     public Context getContext() {
         return defaultContext;
     }
+
+    public ArrayList<RelOptRule> getAllRules() {
+        return this.ALL_RULES;
+    }
+
+    public ArrayList<RelOptRule> getDisabledRules() {
+        return this.disabledRules;
+    }
```

Comment:
Does it make sense to also maintain a list of enabledRules in this class, or is it up to the python layer to compute that with these two bits of information.

Reply:
Yep, that would be really easy to do so I will add it to the Java side. Also I need to actually expose these functions to the python side since I had forgotten to do that in the past commit.

Reply:
@ayushdg would it be helpful if the return value here was just a list of strings instead of a list of `RelOptRule`?